docs/manual/guide/production/ConductR.md

```diff
@@ -1,21 +1,43 @@
 # Lightbend ConductR
 
-ConductR is a container orchestration tool with the main goal of delivering operational productivity. ConductR is also designed to  host your Lagom services with resilience.
+ConductR is a container orchestration tool with the main goal of delivering operational productivity. ConductR is also designed to host your Lagom services with resilience.
 
-ConductR is free for development usage and there is a "sandbox" so that you can run ConductR locally and test your services. Please visit [ConductR's product page](http://lightbend.com/products/conductr) in order to download the sandbox, and also for more information on ConductR in general. If you'd like to know more about our commercial license then [please contact us](https://www.lightbend.com/company/contact). The remainder of this guide will discuss the specific integration points between Lagom and ConductR.
+ConductR is free for development usage and there is a "sandbox" so that you can run ConductR locally and test your services. For more information on ConductR go to:
+
+* [ConductR product page](http://lightbend.com/products/conductr)
+* [ConductR documentation](http://conductr.lightbend.com)
+
+This guide will explain how to:
+
+* Install ConductR sandbox
+* Package Lagom services
+* Start local ConductR cluster
+* Load and run services during and outside of development
+* Run Cassandra on ConductR
+ 
+If you'd like to know more about our commercial license then [please contact us](https://www.lightbend.com/company/contact).
+
+## Installing ConductR sandbox
+
+The ConductR sandbox is a docker image to easily create a ConductR cluster locally. To run ConductR on your development machine follow the instructions on the [ConductR developer page](https://www.lightbend.com/product/conductr/developer) up to the section **Configure Docker Machine**. We will run the sandbox later on in this guide. Note that in order to access this page you need to login with your Lightbend account.
```

Comment:
What I'm trying to hint here is that there is room for confusion. One may understand that you need to be a (paying) customer to have a Lightbend account, so it may stop here without further investigation.